### DISCUSSION TITLE
🚨 Important Update for Contributors 🚨 Slow Down PR's

### DISCUSSION THREAD

**Discussion by MickLesk** [community-scripts/ProxmoxVE#675] [Announcements]
Dear Contributors and all other interested parties,
[@community-scripts/contributor @community-scripts/scripter]

Please **slow down PR submissions** until the following PR is merged:  
🔗 [#643: Critical Update](https://github.com/community-scripts/ProxmoxVE/pull/643)  

---

## 🛠 Guidelines for Now  
To keep our workflow smooth and organized, please follow these temporary rules until the **end of calendar week 50**:  

### 🚦 PR and Dashboard Rules  
1. 🐢 **Slow down PR reviews** until PR #643 is merged.  
2. 🛑 **No new scripts** — Only bug fixes are accepted during this period.  
3. ⚙️ **Fixes first:** Bug fixes (Issues > PR) take priority over new scripts.  
4. ⏳ Expect **delays in PR reviews** while we align our processes.  

### 📋 Project Dashboard Usage  
- Only pick scripts from the **Ready** section in the project dashboard.  
- These are the only scripts currently approved for the pipeline.  

---

## 📝 Looking Ahead  
We are working on a more reliable `CONTRIBUTOR.md` and standardized templates.  
🔧 **Why?**  
The current scripts vary too much in style and structure, making reviews difficult and time-consuming.  
📜 **Goal:** Establish a clear and easy-to-follow standard for everyone.  

Thank you for your understanding and patience while we make improvements! 🙏  

Let’s build something great together! 🚀

**Comment by CrazyWolf13** [comment]
@MickLesk  you are doing an amazing job, thanks a lot for taking over this big role in managin the community and everything else!

**Comment by MickLesk** [comment]
https://github.com/community-scripts/ProxmoxVE/discussions/841

Closing. Done with this Discussion
